### ISSUE TITLE
bash_completion does not install

### ISSUE THREAD

**Issue by peterlynch** [getcarina/dvm#73]
Install using 

```
curl -sL https://download.getcarina.com/dvm/latest/install.sh | sh
```

You end up with this in `.dvm/bash_completion`

```
<html><h1>Not Found</h1><p>The resource could not be found.</p></html>
```

The latest tag does not include bash_completion?

**Comment by rgbkrk** [comment]
Confirmed that. I'll get it fixed.

**Comment by rgbkrk** [comment]
Whoops, meant to post after I fixed this. I've put the bash completion up manually and fixed our automation in #74.
